monte_carlo.py

```diff
@@ -112,13 +120,14 @@ def transport_neutron(mat, bounds, tallies, fission_banks, first_round, mesh):
                 
             # r is a variable that contains the distance along the
             # direction vector to the boundary being tested  
-            
+
             # test each boundary
             for axis in axes:
                 for side in ['min', 'max']:
                     r = distance_to_cell_edge[axis][side] \
                             / neutron.get_direction(axis)
-                    if (r > 0 and r < tempd and r != -0.0):
+                    
+                    if (r > 0 and r < tempd):
```

Comment:
Why did we remove the third check on ``r``? I like this version much better, but have you made sure we still get the correct behavior with the check removed?